### ISSUE TITLE
[Bug] Allow unknown flag is not passed along.

### ISSUE THREAD

**Issue by popescu-af** [stephenafamo/scan#4]
Please see https://github.com/stephenafamo/scan/blob/69030677cb2df82cced579bf55308cb7d89f3b72/row.go#L19

Did a test on a fork and simply passing the flag like
```go
	return &Row{
		r:                r,
		columns:          cols,
		scanDestinations: make([]reflect.Value, len(cols)),
		allowUnknown:     allowUnknown,
	}, nil
```
seems to solve the problem.

Very nice tool BTW, if I manage to integrate it with my SQLC plugin I can wipe a lot of boilerplate out. Thanks!

**Comment by stephenafamo** [comment]
Nice find. Kindly send in a PR with this change along with tests to prevent a regression 🙏🏾

**Comment by popescu-af** [comment]
> Nice find. Kindly send in a PR with this change along with tests to prevent a regression 🙏🏾

Cool, I’ll do it soon. Thanks.

**Comment by popescu-af** [comment]
Sorry for the late reply, I was on vacation. I'd like to create a PR, but I guess I don't have access rights to the repository. If you could give me access, I will create a branch and PR. Thanks!

**Comment by stephenafamo** [comment]
Fixed by #5
